package.json

```diff
@@ -60,6 +60,8 @@
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-react": "^6.3.13",
     "del": "^2.2.0",
+    "grommet-toolbox": "^0.1.14",
```

Comment:
i believe we need to remove grommet-toolbox from here. For now we will leave in dependencies only. When we release grommet 0.6.0 we will move it back to devDependencies.